tests/Unit/Helpers/SSHTest.php

```
@@ -0,0 +1,32 @@
+    use RefreshDatabase;
+
+    public function test_write_removes_the_temporary_file_on_both_the_success_and_failure_branches(): void
+    {
+        SSH::fake();
+
+        $this->server->ssh()->write('/home/vito/example.com/.env', 'APP_NAME=TestApp', 'vito');
+
+        SSH::assertExecutedContains("> '/home/vito/example.com/.env'; then");
+        SSH::assertExecutedContains("else\n    rm -f ");
+        SSH::assertExecutedContains('exit 1');
+    }
```

PHPMD (2.15.0)

[error] The method test_write_removes_the_temporary_file_on_both_the_success_and_failure_branches is not named in camelCase. (undefined)

(CamelCaseMethodName)